website/docs/d/schema_site_anp_epg_bulk_staticport.html.markdown

```diff
@@ -0,0 +1,49 @@
+---
+layout: "mso"
+page_title: "MSO: mso_schema_site_anp_epg_bulk_staticport"
+sidebar_current: "docs-mso-data-source-schema_site_anp_epg_bulk_staticport"
+description: |-
+  Data source for MSO Schema Site ANP EPG Bulk Static Port.
+---
+
+# mso_schema_site_anp_epg_bulk_staticport #
+
+Data source for MSO Schema Site ANP EPG Bulk Static Port.
+
+## Example Usage ##
+
+```hcl
+
+data "mso_schema_site_anp_epg_bulk_staticport" "static_port" {
+  schema_id     = data.mso_schema.schema1.id
+  site_id       = data.mso_site.site1.id
+  template_name = "Template1"
+  anp_name      = "ANP"
+  epg_name      = "DB"
+}
+
+```
+
+## Argument Reference ##
+
+* `schema_id` - (Required) SchemaID under which you want to deploy Static Port.
```

Comment:
I'm not sure if we should use 'you' in the docs. It would be good if we could change this to something like this -
SchemaID under which the Static Port is deployed